presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorMetadata.java

```diff
@@ -692,7 +694,18 @@ public ConnectorInsertTableHandle beginInsert(ConnectorSession session, Connecto
         List<ColumnInfo> columns = handle.getColumnHandles().stream().map(ColumnInfo::fromHandle).collect(toList());
         long updateTime = session.getStartTime();
 
-        shardManager.commitShards(transactionId, tableId, columns, parseFragments(fragments), externalBatchId, updateTime);
+        Collection<ShardInfo> shards = parseFragments(fragments);
+        int tableColumnCount = handle.getColumnHandles().size();
+        log.info("Shards inserted: %s\tCurrent table column count: %s", shards.size(), tableColumnCount);
```

Comment:
Don't use tabs in log messages. Also, we don't need too log the column count, since that's available in the table metadata. Let's format like this:
```java
log.info("Committing insert of %s shards for tableId %s", ...);